From: Jeff Shaw <jeff.shaw@digi.com>
Date: Wed, 5 Dec 2018 11:46:15 +1000
Subject: plugin: increase plugin probe timeout

Increased the timeouts to allow detection of all ports correctly.
Specifically for LE910NA1, LE910C1-EU, LE910C1-AP, and friends.
---
 src/mm-plugin-manager.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/mm-plugin-manager.c b/src/mm-plugin-manager.c
index 9ee4edae..bc005582 100644
--- a/src/mm-plugin-manager.c
+++ b/src/mm-plugin-manager.c
@@ -702,11 +702,11 @@ port_context_new (MMPluginManager *self,
 /* Device context */
 
 /* Time to wait for ports to appear before starting to probe the first one */
-#define MIN_WAIT_TIME_MSECS 1500
+#define MIN_WAIT_TIME_MSECS 7500
 
 /* Time to wait for other ports to appear once the first port is exposed
  * (needs to be > MIN_WAIT_TIME_MSECS!!) */
-#define MIN_PROBING_TIME_MSECS 2500
+#define MIN_PROBING_TIME_MSECS 10000
 
 /* Additional time to wait for other ports to appear after the last port is
  * exposed in the system. */
